src/ngx_lua_resty_lmdb_module.c

```diff
@@ -228,50 +232,39 @@ ngx_lua_resty_lmdb_init_conf(ngx_cycle_t *cycle, void *conf)
 }
 
 
-static ngx_int_t ngx_lua_resty_lmdb_init(ngx_cycle_t *cycle) {
-    /* ngx_lua_resty_lmdb_conf_t *lcf;
-
-    lcf = (ngx_lua_resty_lmdb_conf_t *) ngx_get_conf(cycle->conf_ctx,
-                                                     ngx_lua_resty_lmdb_module); */
-
-    return NGX_OK;
-}
-
-
-static ngx_int_t ngx_lua_resty_lmdb_init_worker(ngx_cycle_t *cycle)
+static ngx_int_t
+ngx_lua_resty_lmdb_open_file(ngx_cycle_t *cycle,
+                             ngx_lua_resty_lmdb_conf_t *lcf,
+                             int clear_secret)
```

Comment:
Let's use `ngx_flag_t` instead of `int` for C code that does not interface with FFI to follow Nginx's conventions.

Reply:
done